src/Compilers/Core/CodeAnalysisTest/DefaultAnalyzerAssemblyLoaderTests.cs

```diff
@@ -28,127 +37,195 @@ public class AssemblyLoadTestFixtureCollection : ICollectionFixture<AssemblyLoad
         private AssemblyLoadTestFixtureCollection() { }
     }
 
-    [Collection(AssemblyLoadTestFixtureCollection.Name)]
     public sealed class DefaultAnalyzerAssemblyLoaderTests : TestBase
     {
+#if NETCOREAPP
+        private sealed class AssemblyLoadContextInvokeUtil
```

Comment:
Could we remove this type and use `AppDomainInvokeUtil` for both platforms instead?